packages/storybook/examples/HeaderRow/OptionalArea.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import HeaderRow from '@ichef/gypcrete/src/HeaderRow';
+import Button from '@ichef/gypcrete/src/Button';
+import TextLabel from '@ichef/gypcrete/src/TextLabel';
+
+import DebugBox from 'utils/DebugBox';
+
+function OptionalArea() {
```

Comment:
Could it be renamed as `HeaderWithBlankArea`, or change the description of the story (`Optional area`)? Because at the first glance, I thought there are other additional areas on the `<HeaderRow>`

Reply:
It's not really *blank* area though, it's **removing** entire area `<div>` from DOM.
Let me think of another name for that.